lisa/operating_system.py

```diff
@@ -1814,6 +1814,35 @@ def _update_packages(self, packages: Optional[List[str]] = None) -> None:
         if self._no_repo_enabled.search(output):
             raise RepoNotExistException(self._node.os)
 
+    def grow_os_partition_size(self) -> None:
```

Comment:
Move the methods to tools, instead of in os class.

Reply:
Create a new tool? or have it as a function inside Hibernation feature itself?

Reply:
Yes, you can put it in test case firstly. If other places need it, then refactoring and find a better place.

Reply:
updated